var/spack/repos/builtin/packages/w3nco/package.py

```diff
@@ -23,5 +23,4 @@ class W3nco(CMakePackage):
 
     version('2.4.1', sha256='48b06e0ea21d3d0fd5d5c4e7eb50b081402567c1bff6c4abf4fd4f3669070139')
 
-    if sys.platform == 'darwin' and macos_version() >= Version('12.0'):
-        patch('darwin/apple-clang-13.0.0-times.patch')
+    patch('darwin/apple-clang-13.0.0-times.patch', when="%apple-clang platform=darwin")
```

Comment:
We need to check if the patch is needed for llvm-clang as well. I will know later today.

Reply:
The patch is not required for llvm-clang. `spack install w3nco%clang@13.0.0` worked with no patches.

Reply:
w3nco built fine on my mac with llvm-clang with the original lines 26 and 27, so I guess that patch is either needed or doesn't hurt with llvm-clang. Can you chance it so that the patch gets applied for both?

Reply:
Overlapping conversation. Then it's fine. I can give it another try on my Mac later.